rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
[Optional]: Nicely done! I can see that Getters and Setters for @book and @person instance variables are present. Can you use the[ shortcuts ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor%20#%20*Check%20for%20a%20setter%20and%20getter%20for%20@label)`attr_accessor :date, :book, :person`?  Because these variables are both getters and setters